src/EnergyPlus/TarcogShading.cc

```diff
@@ -588,6 +588,7 @@ namespace TarcogShading {
 		Real64 const s1_2( pow_2( s1 ) );
 		Real64 const s2_2( pow_2( s2 ) );
 		Real64 const s1_s2_2( pow_2( s1 / s2 ) );
+		Real64 const CosTilt = std::cos( tilt );
```

Comment:
I've been trying to keep it consistent by going with ```Real64 cos_foo( std::cos( foo ) );```. I just missed this one. Again, and preference, @Myoldmopar ?